test/Runner.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { Runner } from "../src/Runner";
+
+describe("Runner", () => {
+  class CustomRunner extends Runner {
+    public stdout: string = "";
+    public stderr: string = "";
+    public exitCode: number;
+    protected readFromStdin() {
+      return Promise.resolve("");
+    }
+    protected writeOut(text: string) {
+      this.stdout += text;
+    }
+    protected writeError(text: string) {
+      this.stderr += text;
+    }
+    protected exit(exitCode: number) {
+      this.exitCode = exitCode;
+    }
+    public toJSON(): object {
+      return {
+        exitCode: this.exitCode,
+        stderr: this.stderr,
+        stdout: this.stdout,
+      };
+    }
+  }
+  describe("Support", () => {
+    describe("with JSON", () => {
+      test("languages", () => {
+        const runner = new CustomRunner();
+        runner.support({
```

Comment:
This is async now, right? You may need to `await`.
And make sure to either return a Promise or use some other method to let Jest know this test is async.

Reply:
I needed `async` in both the `describe` and `test` blocks.  I had it only in the `test` block before.